### DISCUSSION TITLE
Move mouse between screens

### DISCUSSION THREAD

**Discussion by KvelaGorrrrnio** [rxhanson/Rectangle#597] [Ideas]
I often work on 2 screens and being able to move windows between them with rectangle is great!
A feature that I miss is the ability to move only the cursor from one screen to another. E.g. after moving a window from screen A to B, I need to move the cursor from B to A.

**Comment by rxhanson** [comment]
Are you asking for a keyboard shortcut to move the cursor to a different display?

**Reply by KvelaGorrrrnio** [reply]
Yeah. I don't know if it is outside of the scope of what Rectangle should be, but it would make organizing windows across multiple workspaces and screens a lot easier.

**Reply by rxhanson** [reply]
It’s a bit out of the scope of what I would add in, but I’d be willing to consider a pr for it as long as it is only a terminal config and not in the UI.

**Comment by thomasfr** [comment]
i would love to have that in rectangle too. there are standalone apps which can do that but i would highly prefer to have this integrated in rectangle directly

**Reply by rxhanson** [reply]
Rectangle is purposefully slim, but I would consider adding this into one of my other apps, Rectangle Pro or Multitouch. Where would you want your cursor to go? Center of display, corner?

**Reply by thomasfr** [reply]
yes, i like that its slim too. Would love to see it in Rectangle Pro though. center of display or same relative position as it is on the current display. but i assume center is sufficient

**Reply by beddari** [reply]
Probably it would be best to move the pointer to where the current keyboard focus  / caret is?

**Reply by beddari** [reply]
That way when I move my apps around my 4x displays it would be a lot faster to follow up working where I moved to. Also, I am actually using two keyboards quite often (standing/walking desk setup, and the laptop keyboard while resting. on the side) :-)

**Reply by beddari** [reply]
Hmm, someone said this is already somewhat possible? https://github.com/rxhanson/Rectangle/discussions/794#discussioncomment-3796486

**Reply by beddari** [reply]
Updated, and found it. Sorry for the noise here. Bought Pro to support even if I don't need it.
